### PR TITLE
Completed Express Post JSON

### PR DESCRIPTION
**Pull request by CollinHovey**
Code_Pu0okWPoPI

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }
+  res.json(sendGrades);
+});
+
+app.post('/api/grades', (req, res) => {
+  const newStudent = req.body;
+  newStudent.id = nextId;
+  grades[nextId] = newStudent;
+  nextId += 1;
+  res.json(newStudent);
```
Comment (scbowler) [inline express-post-json/index.js:22]: Here you want to set the response status code to `201`:

```JAVASCRIPT
res.status(201).json(newStudent);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but 1 update needed. Let me know if you have any questions.

**Comment by CollinHovey** [comment]
Code_mjIGhA3nRY

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**